flycheck.el

```diff
@@ -7512,6 +7512,20 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((command (concat (flycheck-find-checker-executable checker)
+                                     (if flycheck-eslintrc
```

Comment:
Oh, I didn't know that we still had an option for eslint's configuration file.  I think we should remove it, shouldn't we?  After all, eslint does a much better job at finding its own configuration than Flycheck, doesn't it?